### DISCUSSION TITLE
我的路由器是XT8，刷了G大的梅林固件，應該如何安裝？

### DISCUSSION THREAD

**Discussion by stebrick** [messense/aliyundrive-webdav#562] [Q&A] [answered]
小白求教。
非koolcenter固件，只刷了gnuton大神的梅林固件(386.07_2-gnuton)，應該如何用ssh安裝？最主要是不知道安裝哪幾個檔案(目測是armv7?)、安裝命令以及安裝的目錄，還有安裝後的後續設定及如何調出refresh token等。搜索了半天無果，求大神打救，感謝萬分。

**Comment by messense** [comment, marked as answer]
不支持非 koolshare 版本梅林固件。
